src/Ilios/CoreBundle/Entity/Repository/MeshDescriptorRepository.php

```diff
@@ -28,19 +28,20 @@ class MeshDescriptorRepository extends EntityRepository implements DTORepository
      */
     public function findByQ($q, $orderBy, $limit, $offset)
     {
+        $terms = explode(' ', $q);
+        $terms = array_filter($terms, 'strlen');
+        if (empty($terms)) {
+            return [];
+        }
+
         $qb = $this->_em->createQueryBuilder()
             ->select('DISTINCT d')
             ->from('IliosCoreBundle:MeshDescriptor', 'd')
             ->leftJoin('d.previousIndexing', 'pi')
             ->leftJoin('d.concepts', 'c')
             ->leftJoin('c.semanticTypes', 'st')
-            ->leftJoin('c.terms', 't');
-
-        $terms = explode(' ', $q);
-        $terms = array_filter($terms, 'strlen');
-        if (empty($terms)) {
-            return [];
-        }
+            ->leftJoin('c.terms', 't')
+            ->where('d.deleted = false');
```

Comment:
Usually a good idea to must make this `andWhere`, prevent issues with chaining and ordering in the future.